dd-sdk-android/src/test/kotlin/com/datadog/android/rum/internal/domain/scope/RumViewScopeTest.kt

```diff
@@ -836,28 +1095,75 @@ internal class RumViewScopeTest {
     }
 
     @Test
-    fun `unknown events are sent to children ResourceScopes`(
+    fun `𝕄 do nothing 𝕎 handleEvent(StartAction) on stopped view`(
+        @StringForgery(StringForgeryType.ALPHABETICAL) key: String,
+        @StringForgery(StringForgeryType.ALPHABETICAL) method: String,
+        @RegexForgery("http(s?)://[a-z]+.com/[a-z]+") url: String,
+        forge: Forge
+    ) {
+        // Given
+        val attributes = forge.exhaustiveAttributes()
+        testedScope.stopped = true
+
+        // When
+        val result = testedScope.handleEvent(
+            RumRawEvent.StartResource(key, url, method, attributes),
```

Comment:
shouldn't be a `StartAction` event here instead ?

Reply:
Indeed it's a bad copy paste. The actual method already exists above :/